backend/api/views.py

```diff
@@ -181,3 +182,24 @@ def post(self, request):
             return Response({"sac_id": sac.id, "next": "TBD"})
 
         return Response({"errors": serializer.errors})
+
+
+class SubmissionsView(APIView):
+    """
+    Returns the list of SingleAuditChecklists the current user has submitted
+    """
+
+    def get(self, request):
+        current_user = request.user
+
+        all_submissions = SingleAuditChecklist.objects.filter(
+            submitted_by=current_user
+        ).values(
+            "report_id",
+            "submission_status",
+            "auditee_uei",
+            "auditee_fiscal_period_end",
+            "auditee_name",
+        )
+
+        return JsonResponse(list(all_submissions), safe=False)
```

Comment:
[NB] Sweet, you figured out how to do this right!

Reply:
Only with your help!!! 

![teamwork](https://media.giphy.com/media/3oEjHV0z8S7WM4MwnK/giphy.gif)